_templates/sidebar_links.html

```diff
@@ -10,27 +10,23 @@
   <script type="text/javascript">
     nbviewer_random_thumbnail();
   </script>
-	       
-</div> 
+
+</div>
 </div>
 
 <div class="sidebarblock">
 <h3>{{ _('Community') }}</h3>
 
   <ul class="simple">
-    <li><a class="reference external" href="https://gitter.im/ipython/ipython/help">
-            Help Chat Room</a></li>
-    <li><a class="reference external" href="http://stackoverflow.com/questions/tagged/ipython">
```

Comment:
Agree on leaving Stack Overflow, even though it's not focused content. The nice thing there is that when we answer questions it helps a lot more people.

We're operating enough stuff, so if you don't want to moderate forums I wouldn't start a discourse right away. Not to say it wouldn't be useful, just that we'd probably want decent moderators for it.